addons/base/models.py

```diff
@@ -108,7 +108,7 @@ def nodes_authorized(self):
         model = self.config.node_settings
         if not model:
             return []
-        return [obj.owner for obj in model.objects.filter(user_settings=self, owner__is_deleted=False)]
+        return [obj for obj in model.objects.filter(user_settings=self, owner__is_deleted=False).select_related('owner')]
```

Comment:
You will still need to do `obj.owner` if you want to return the nodes rather than the node settings objects.